glean-core/android/build.gradle

```diff
@@ -74,15 +75,21 @@ cargo {
     // The Cargo targets to invoke.  The mapping from short name to target
     // triple is defined by the `rust-android-gradle` plugin.
     targets = [
-        //'linux-x86-64',
-        //'darwin',
-        //'win32-x86-64-gnu',
         'arm',
         'arm64',
         'x86',
         'x86_64',
     ]
 
+    // Build for the "host" platform
+    if (Os.isFamily(Os.FAMILY_WINDOWS)) {
+        targets.push('win32-x86-64-gnu')
+    } else if (Os.isFamily(Os.FAMILY_MAC)) {
+        targets.push('darwin')
+    } else if (Os.isFamily(Os.FAMILY_UNIX)) {
+        targets.push('linux-x86-64')
+    }
```

Comment:
We should each test this to make sure it's working everywhere.

Reply:
Can we file an issue to check if the plugin can handle that?

Reply:
Looks like there's already an open issue for this [here](https://github.com/ncalexan/rust-android-gradle/issues/33).